### PR TITLE
wip

### PR DESCRIPTION
**Pull request by sidvishnoi**
testing pr...
add comments in this PR to test how the comment toggle works and indicator works

### CODE REVIEW

File: source/features/toggle-review-comments.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import features from "../libs/features";
+import { comment as commentIcon } from "../libs/icons";
+
+let isVisible = true;
+
+function toggleComments(event: Event) {
+	const togglers = select.all(".js-toggle-file-notes") as HTMLInputElement[];
+
+	let scrollOffset = window.scrollY;
+	// const target = event.target as HTMLElement;
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment 1

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: comment 2

File: source/features/toggle-review-comments.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	// const nearestContainer = target.closest(".inline-comments")!;
+	// const containers = select.all("tr.inline-comments");
+	// for (
+	// 	let i = 0;
+	// 	i < containers.length && nearestContainer === containers[i];
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: new comment

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: lalaa

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: foo

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: foo2

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: foo 3

File: source/features/toggle-review-comments.tsx (hunk 1 above)
Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: aaa

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: bbbb

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: ccc

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: ddd

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: eee

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: foo

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: bar

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 4

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 5

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 3

File: source/features/toggle-review-comments.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import "./toggle-review-comments.css";
+import React from "dom-chef";
+import select from "select-dom";
+// import delegate from "delegate-it";
+import features from "../libs/features";
+import { comment as commentIcon } from "../libs/icons";
+
+let isVisible = true;
+
+function toggleComments(event: Event) {
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import "./toggle-review-comments.css";
+import React from "dom-chef";
+import select from "select-dom";
+// import delegate from "delegate-it";
+import features from "../libs/features";
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: new new

File: source/features/toggle-review-comments.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
+import "./toggle-review-comments.css";
+import React from "dom-chef";
+import select from "select-dom";
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: new new

Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: NEW NEW

Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: new comment

Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+// import delegate from "delegate-it";
+import features from "../libs/features";
+import { comment as commentIcon } from "../libs/icons";
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+let isVisible = true;
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx (hunk 5 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+	// const containers = select.all("tr.inline-comments");
+	// for (
+	// 	let i = 0;
+	// 	i < containers.length && nearestContainer === containers[i];
+	// 	i++
+	// ) {
+	// 	scrollOffset -= containers[i].offsetHeight;
+	// }
+
+	togglers.forEach(input => {
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 9]
```diff
@@ -0,0 +1,73 @@
+import "./toggle-review-comments.css";
+import React from "dom-chef";
+import select from "select-dom";
+// import delegate from "delegate-it";
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: foo

File: source/features/toggle-review-comments.tsx (hunk 5 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: foo

File: source/features/toggle-review-comments.tsx (hunk 9 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: foo

Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: init?

File: source/features/toggle-review-comments.tsx [hunk 10]
```diff
@@ -0,0 +1,73 @@
+import "./toggle-review-comments.css";
+import React from "dom-chef";
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: foo

Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: asdasd

File: source/features/toggle-review-comments.tsx (hunk 7 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: foo

File: source/features/toggle-review-comments.tsx [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+	// for (
+	// 	let i = 0;
+	// 	i < containers.length && nearestContainer === containers[i];
+	// 	i++
+	// ) {
+	// 	scrollOffset -= containers[i].offsetHeight;
+	// }
+
+	togglers.forEach(input => {
+		if (isVisible !== input.checked) {
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+			input.click();
+		}
+	});
+	isVisible = !isVisible;
+
+	window.scrollTo(0, scrollOffset);
+}
+
+function addButtons(): void {
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+			"afterbegin",
+			<td className="blob-num refined-toggle-comments" colspan="2">
+				<button onClick={toggleComments}>
+					{commentIcon()}
+					<span>{commentCount}</span>
+				</button>
+			</td>,
+		);
+	});
+}
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+
+function init(): void {
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+
+function addButtons(): void {
+	const commentsContainers = select.all("tr.inline-comments");
+	commentsContainers.forEach(container => {
+		const commentCount = container.querySelectorAll(".review-comment").length;
+		container.insertAdjacentElement(
+			"afterbegin",
+			<td className="blob-num refined-toggle-comments" colspan="2">
+				<button onClick={toggleComments}>
+					{commentIcon()}
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: casdas

File: source/features/toggle-review-comments.tsx (hunk 14 above)
Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: sdfsdf

File: source/features/toggle-review-comments.tsx [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+function init(): void {
+	console.log("init");
+	addButtons();
+
+	// const repoContent = select(".repository-content")!;
+	// observeEl(repoContent, addButton);
+	// delegate(".rgh-toggle-files", "click", ({ delegateTarget }) => {
+	// 	delegateTarget.setAttribute(
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: dfsdfsd

File: source/features/toggle-review-comments.tsx [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+	// 		"aria-expanded",
+	// 		String(!repoContent.classList.toggle("rgh-files-hidden")),
+	// 	);
+	// });
+}
+
+features.add({
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: asdasd

File: source/features/toggle-review-comments.tsx [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+	// 	i++
+	// ) {
+	// 	scrollOffset -= containers[i].offsetHeight;
+	// }
+
+	togglers.forEach(input => {
+		if (isVisible !== input.checked) {
+			input.click();
+		}
+	});
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: fsdfsd

File: source/features/toggle-review-comments.tsx [hunk 19]
```diff
@@ -0,0 +1,73 @@
...
+	// 		"aria-expanded",
+	// 		String(!repoContent.classList.toggle("rgh-files-hidden")),
+	// 	);
+	// });
+}
+
+features.add({
+	id: __featureName__,
+	description: "Adds comment indicators when comments are hidden on PR review",
+	screenshot:
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sfsdfsd

File: source/features/toggle-review-comments.tsx [hunk 20]
```diff
@@ -0,0 +1,73 @@
...
+	console.log("init");
+	addButtons();
+
+	// const repoContent = select(".repository-content")!;
+	// observeEl(repoContent, addButton);
+	// delegate(".rgh-toggle-files", "click", ({ delegateTarget }) => {
+	// 	delegateTarget.setAttribute(
+	// 		"aria-expanded",
+	// 		String(!repoContent.classList.toggle("rgh-files-hidden")),
+	// 	);
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: asdsd

File: source/features/toggle-review-comments.tsx [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+function addButtons(): void {
+	const commentsContainers = select.all("tr.inline-comments");
+	commentsContainers.forEach(container => {
+		const commentCount = container.querySelectorAll(".review-comment").length;
+		container.insertAdjacentElement(
+			"afterbegin",
+			<td className="blob-num refined-toggle-comments" colspan="2">
+				<button onClick={toggleComments}>
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sfsdf

File: source/features/toggle-review-comments.tsx (hunk 3 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: comment

File: source/features/toggle-review-comments.tsx [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+	});
+}
+
+function init(): void {
+	console.log("init");
+	addButtons();
+
+	// const repoContent = select(".repository-content")!;
+	// observeEl(repoContent, addButton);
+	// delegate(".rgh-toggle-files", "click", ({ delegateTarget }) => {
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfasd

File: source/features/toggle-review-comments.tsx (hunk 16 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: zcsddfsd

File: source/features/toggle-review-comments.tsx [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+		);
+	});
+}
+
+function init(): void {
+	console.log("init");
+	addButtons();
+
+	// const repoContent = select(".repository-content")!;
+	// observeEl(repoContent, addButton);
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsf

File: source/features/toggle-review-comments.tsx [hunk 24]
```diff
@@ -0,0 +1,73 @@
...
+	let scrollOffset = window.scrollY;
+	// const target = event.target as HTMLElement;
+	// const nearestContainer = target.closest(".inline-comments")!;
+	// const containers = select.all("tr.inline-comments");
+	// for (
+	// 	let i = 0;
+	// 	i < containers.length && nearestContainer === containers[i];
+	// 	i++
+	// ) {
+	// 	scrollOffset -= containers[i].offsetHeight;
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdf

File: source/features/toggle-review-comments.tsx (hunk 18 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdf

File: source/features/toggle-review-comments.tsx [hunk 25]
```diff
@@ -0,0 +1,73 @@
...
+	// }
+
+	togglers.forEach(input => {
+		if (isVisible !== input.checked) {
+			input.click();
+		}
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdfsdf

File: source/features/toggle-review-comments.tsx (hunk 8 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdfsdf

File: source/features/toggle-review-comments.tsx [hunk 26]
```diff
@@ -0,0 +1,73 @@
...
+	// const nearestContainer = target.closest(".inline-comments")!;
+	// const containers = select.all("tr.inline-comments");
+	// for (
+	// 	let i = 0;
+	// 	i < containers.length && nearestContainer === containers[i];
+	// 	i++
+	// ) {
+	// 	scrollOffset -= containers[i].offsetHeight;
+	// }
+
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdf

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: sdfsdfsdf

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: sdfsdfsdf

File: source/features/toggle-review-comments.tsx [hunk 27]
```diff
@@ -0,0 +1,73 @@
...
+	togglers.forEach(input => {
+		if (isVisible !== input.checked) {
+			input.click();
+		}
+	});
+	isVisible = !isVisible;
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdfsdf

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: sdfsdfsdf

File: source/features/toggle-review-comments.tsx [hunk 28]
```diff
@@ -0,0 +1,73 @@
+import "./toggle-review-comments.css";
+import React from "dom-chef";
+import select from "select-dom";
+// import delegate from "delegate-it";
+import features from "../libs/features";
+import { comment as commentIcon } from "../libs/icons";
+
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: fsdfsddf

File: source/features/toggle-review-comments.tsx [hunk 29]
```diff
@@ -0,0 +1,73 @@
...
+let isVisible = true;
+
+function toggleComments(event: Event) {
+	const togglers = select.all(".js-toggle-file-notes") as HTMLInputElement[];
+
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: dfgdfgdfg

File: source/features/toggle-review-comments.tsx [hunk 30]
```diff
@@ -0,0 +1,73 @@
...
+			input.click();
+		}
+	});
+	isVisible = !isVisible;
+
+	window.scrollTo(0, scrollOffset);
+}
+
+function addButtons(): void {
+	const commentsContainers = select.all("tr.inline-comments");
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsddfsdf

File: source/features/toggle-review-comments.tsx (hunk 21 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: dfgdfgdfg

File: source/features/toggle-review-comments.tsx (hunk 14 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdfsdf

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: sfsddfsdf

File: source/features/toggle-review-comments.tsx [hunk 31]
```diff
@@ -0,0 +1,73 @@
...
+	commentsContainers.forEach(container => {
+		const commentCount = container.querySelectorAll(".review-comment").length;
+		container.insertAdjacentElement(
+			"afterbegin",
+			<td className="blob-num refined-toggle-comments" colspan="2">
+				<button onClick={toggleComments}>
+					{commentIcon()}
+					<span>{commentCount}</span>
+				</button>
+			</td>,
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdf

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: sdfsdfsdf

File: source/features/toggle-review-comments.tsx [hunk 32]
```diff
@@ -0,0 +1,73 @@
...
+
+function init(): void {
+	console.log("init");
+	addButtons();
+
+	// const repoContent = select(".repository-content")!;
+	// observeEl(repoContent, addButton);
+	// delegate(".rgh-toggle-files", "click", ({ delegateTarget }) => {
+	// 	delegateTarget.setAttribute(
+	// 		"aria-expanded",
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: 1

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 2

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 3

File: source/features/toggle-review-comments.tsx (hunk 28 above)
Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: fsdfsdfsd

File: source/features/toggle-review-comments.tsx (hunk 12 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: 1

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 2

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 3

File: source/features/toggle-review-comments.tsx (hunk 26 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: 1

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: 2

File: source/features/toggle-review-comments.tsx (hunk 14 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: 1

File: source/features/toggle-review-comments.tsx (hunk 7 above)
Comment (sidvishnoi) [inline source/features/toggle-review-comments.tsx]: sdfsdf

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: dfsdfsdf

Comment (sidvishnoi) [reply source/features/toggle-review-comments.tsx]: sdfsdfds

File: source/features/toggle-review-comments.css [hunk 33]
```diff
@@ -0,0 +1,37 @@
+tr.refined-toggle-comments {
+	position: relative;
+}
+tr.refined-toggle-comments td {
```
Comment (sidvishnoi) [inline source/features/toggle-review-comments.css]: sdfsdf

File: source/features/review-comments-hidden-indicator.tsx [hunk 34]
```diff
@@ -0,0 +1,112 @@
...
+	const commentCount = select.all(SELECTOR_COMMENT, container).length;
+	if (!commentCount) return;
+	container.before(CommentToggle(commentCount));
+	observeEl(container, countListener, { childList: true, subtree: true });
+};
+
+function init() {
+	select.all(SELECTOR_COMMENT_CONTAINER).forEach(el => addToggle(el));
+
+	// watching for addition/deletion of comment containers
```
Comment (sidvishnoi) [inline source/features/review-comments-hidden-indicator.tsx]: this is a comment

Comment (sidvishnoi) [reply source/features/review-comments-hidden-indicator.tsx]: and this is a reply

File: source/features/review-comments-hidden-indicator.tsx [hunk 35]
```diff
@@ -0,0 +1,112 @@
...
+	select.all('.diff-table tbody').forEach(tbody => {
+		observeEl(tbody, onTableMutation, { childList: true, subtree: true });
+	});
+}
+
+features.add({
+	id: __featureName__,
+	description: 'Adds comment indicators when comments are hidden in PR review',
+	screenshot:
+		'https://user-images.githubusercontent.com/1402241/35480123-68b9af1a-043a-11e8-8934-3ead3cff8328.gif', // TODO
```
Comment (sidvishnoi) [inline source/features/review-comments-hidden-indicator.tsx]: need to add a gif once finalized

File: source/features/review-comments-hidden-indicator.tsx [hunk 36]
```diff
@@ -0,0 +1,112 @@
...
+			}
+		}
+	};
+	select.all('.diff-table tbody').forEach(tbody => {
+		observeEl(tbody, onTableMutation, { childList: true, subtree: true });
+	});
+}
+
+features.add({
+	id: __featureName__,
```
Comment (sidvishnoi) [inline source/features/review-comments-hidden-indicator.tsx]: a better feature name is welcome

File: source/features/review-comments-hidden-indicator.tsx [hunk 37]
```diff
@@ -0,0 +1,112 @@
+import './review-comments-hidden-indicator.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import observeEl from '../libs/simplified-element-observer';
+import { comment as commentIcon } from '../libs/icons';
```
Comment (sidvishnoi) [inline source/features/review-comments-hidden-indicator.tsx]: commentIcon taken from GitHub issues page 🎉

File: source/features/review-comments-hidden-indicator.tsx (hunk 36 above)
Comment (sidvishnoi) [reply source/features/review-comments-hidden-indicator.tsx]: also a better description

File: source/content.ts [hunk 38]
```diff
@@ -19,6 +19,7 @@ import './features/sticky-discussion-list-toolbar.css';
 // DO NOT add CSS files here if they are part of a JavaScript feature.
 // Import the `.css` file from the `.tsx` instead.
 
+import './features/review-comments-hidden-indicator';
```
Comment (sidvishnoi) [inline source/content.ts]: another comment

File: source/features/review-comments-hidden-indicator.tsx [hunk 39]
```diff
@@ -0,0 +1,131 @@
+import './review-comments-hidden-indicator.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import observeEl from '../libs/simplified-element-observer';
+import {comment as commentIcon} from '../libs/icons';
```
Comment (sidvishnoi) [inline source/features/review-comments-hidden-indicator.tsx]: commentIcon from GitHub issues page

File: source/content.ts [hunk 40]
```diff
@@ -19,6 +19,7 @@ import './features/sticky-discussion-list-toolbar.css';
...
 import './features/useful-not-found-page';
 import './features/trending-menu-item';
```
Comment (fregante) [inline source/content.ts]: Same file comment

File: source/content.ts (hunk 38 above)
Comment (fregante) [inline source/content.ts]: Another thread on the same line. It's possible

File: source/libs/icons.tsx [hunk 41]
```diff
@@ -40,6 +40,8 @@ export const chevronLeft = (): SVGElement => <svg aria-hidden="true" className="
 
 export const clock = (): SVGElement => <svg aria-hidden="true" className="octicon octicon-clock" width="14" height="16"><path d="M8 8h3v2H7c-.55 0-1-.45-1-1V4h2v4zM7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7z"/></svg>;
 
+export const comment = (): SVGElement => <svg className="octicon octicon-comment" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M14 1H2c-.55 0-1 .45-1 1v8c0 .55.45 1 1 1h2v3.5L7.5 11H14c.55 0 1-.45 1-1V2c0-.55-.45-1-1-1zm0 9H7l-2 2v-2H2V2h12v8z"/></svg>;
```
Comment (sidvishnoi) [inline source/libs/icons.tsx]: test comment

File: source/features/review-comments-hidden-indicator.css [hunk 42]
```diff
@@ -0,0 +1,37 @@
...
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 200%;
+}
+
+.show-inline-notes tr.refined-toggle-comments td::after {
+	background: transparent;
+}
+
```
Comment (sidvishnoi) [inline source/features/review-comments-hidden-indicator.css]: new comment. clicking the blue add comment button triggered mutation observer to show that this file just got added `.has-inline-notes`.

**Comment by sidvishnoi** [comment]
@fregante would suggest you to unsubscribe this PR. I'm using this as a playground 😅.

File: source/libs/icons.tsx (hunk 41 above)
Comment (sidvishnoi) [inline source/libs/icons.tsx]: test comment addition

Comment (sidvishnoi) [inline source/libs/icons.tsx]: add another comment on same line

File: source/libs/icons.tsx [hunk 43]
```diff
@@ -40,6 +40,8 @@ export const chevronLeft = (): SVGElement => <svg aria-hidden="true" className="
...
+
 export const commit = (): SVGElement => <svg aria-hidden="true" className="octicon octicon-git-commit" width="14" height="16" ><path d="M10.86 7c-.45-1.72-2-3-3.86-3-1.86 0-3.41 1.28-3.86 3H0v2h3.14c.45 1.72 2 3 3.86 3 1.86 0 3.41-1.28 3.86-3H14V7h-3.14zM7 10.2c-1.22 0-2.2-.98-2.2-2.2 0-1.22.98-2.2 2.2-2.2 1.22 0 2.2.98 2.2 2.2 0 1.22-.98 2.2-2.2 2.2z"/></svg>;
 
 export const diff = (): SVGElement => <svg aria-hidden="true" className="octicon octicon-diff" width="13" height="16"><path d="M6 7h2v1H6v2H5V8H3V7h2V5h1v2zm-3 6h5v-1H3v1zM7.5 2L11 5.5V15c0 .55-.45 1-1 1H1c-.55 0-1-.45-1-1V3c0-.55.45-1 1-1h6.5zM10 6L7 3H1v12h9V6zM8.5 0H3v1h5l4 4v8h1V4.5L8.5 0z"/></svg>;
```
Comment (sidvishnoi) [inline source/libs/icons.tsx]: one more comment

**Pull request closed without merging**